Makefile

```diff
@@ -17,5 +17,9 @@ README.html: README.md docs/metadata.yml
 	docs/metadata.yml README.md -o $@ \
 	--standalone --toc --toc-depth 2
 
+Documentation.txt: README.md docs/metadata.yml roles/*
```

Comment:
I don't understand why we're including `README.md` in another document.  I don't think we need to duplicate the content.